src/components/LaunchFilter/actions/actionCreators.ts

```diff
@@ -0,0 +1,22 @@
+import { AxiosError } from 'axios';
+import type { Launch } from '../../../shared/api/types/Launch';
+import { LaunchFilters } from '../constants/filerConstants';
+import { LaunchActions } from './actionTypes';
+
+export const requesAction = (filterValue: LaunchFilters) => ({
+  type: `${filterValue}_${LaunchActions.LAUNCH_REQUEST}`,
+  payload: { filterValue },
+});
+
+export const successAction = (
+  filterValue: LaunchFilters,
+  launches: Launch[],
+) => ({
+  type: `${filterValue}_${LaunchActions.LAUNCH_SUCCESS}`,
+  payload: { filterValue: filterValue, [filterValue]: launches },
+});
+
+export const errorAction = (filterValue: LaunchFilters, error: AxiosError) => ({
+  type: `${filterValue}_${LaunchActions.LAUNCH_ERROR}`,
+  payload: { filterValue: filterValue, error: error as AxiosError },
+});
```

Comment:
1. Use convenient name all action creators
2. you don need to pass filterValue as parameter
3. Why do you need as AxiosError?

Reply:
1. Changed to filterRequestAction,...
2. Returning action type consists of filterValue and action type from LaunchActions enum. And I need filterValue to put it into the payload object for creating action type names in the reducer